app/routers/files.py

```
@@ -1778,24 +1861,38 @@ async def finalize_upload(
-                    file_data = final_path.read_bytes()
-                    
-                    # Encrypt the data
-                    encrypted_data, session_key, session_iv = encrypt_session_data(file_data)
+                    # Run zero-memory disk-to-disk streaming encryption
+                    # This replaces the unencrypted file with the encrypted one without loading it in memory
+                    temp_enc = final_path.with_suffix('.enc.tmp')
+                    from app.core.aes_utils import encrypt_file_to_file_streaming
+                    metadata = encrypt_file_to_file_streaming(
+                        str(final_path),
+                        str(temp_enc),
+                        chunk_size=STREAM_BUFFER_SIZE
```

Ruff (0.16.0)

[error] Undefined name `STREAM_BUFFER_SIZE`

(F821)